codegen/model/chart.go

```diff
@@ -51,11 +51,14 @@ type Operator struct {
 // values for Deployment template
 type Deployment struct {
 	// use a DaemonSet instead of a Deployment
-	UseDaemonSet bool                     `json:"-"`
-	Labels       map[string]string        `json:"extraLabels,omitempty"`
-	Annotations  map[string]string        `json:"extraAnnotations,omitempty"`
-	Image        Image                    `json:"image,omitempty"`
-	Resources    *v1.ResourceRequirements `json:"resources,omitempty"`
+	UseDaemonSet    bool                     `json:"-"`
+	PodLabels       map[string]string        `json:"extraPodLabels,omitempty"`
```

Comment:
Why name the yaml field differently? Can we instead keep the names of the struct fields consistent with those of the yaml fields?